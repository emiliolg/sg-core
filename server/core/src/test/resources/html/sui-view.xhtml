<html sui-view="tekgenesis.service.html.Views" prefix="og: http://ogp.me/ns#">
    <head>
    </head>
    <body>
    </body>
</html>
