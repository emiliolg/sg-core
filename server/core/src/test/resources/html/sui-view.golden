<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns#">
    <head>
    </head>
    <body>
    </body>
</html>